src/pages/api/paymentTxParams/[uuid].ts

```diff
@@ -1,8 +1,10 @@
 import { NextApiRequest, NextApiResponse } from 'next';
 import { getPrismaClient } from '@helpers/database';
-import { formatTxDataResponse } from '@/helpers/formatTxDataResponse';
 import { ethers } from 'ethers';
-import { formatTxMessageResponse } from '@/helpers/formatTxMessageResponse';
+
+// TODO: Amhed: Discuss w/Justin if worth to remove
```

Comment:
@justin-lee-cb

Reply:
These can be removed, I can remove them the next time I work on the repo to get a /submitSliceTx endpoint working

Reply:
👍